users/api/metrics.go

```diff
@@ -64,3 +64,30 @@ func (ms *metricsMiddleware) UserInfo(ctx context.Context, key string) (users.Us
 
 	return ms.svc.UserInfo(ctx, key)
 }
+
+func (ms *metricsMiddleware) GenerateResetToken(ctx context.Context, email, host string) error {
+	defer func(begin time.Time) {
+		ms.counter.With("method", "generate_reset_token").Add(1)
+		ms.latency.With("method", "generate_reset_token").Observe(time.Since(begin).Seconds())
+	}(time.Now())
+
+	return ms.svc.GenerateResetToken(ctx, email, host)
+}
+
+func (ms *metricsMiddleware) UpdatePassword(ctx context.Context, email, password string) error {
+	defer func(begin time.Time) {
+		ms.counter.With("method", "change_password").Add(1)
+		ms.latency.With("method", "change_password").Observe(time.Since(begin).Seconds())
```

Comment:
Use `update_password` instead of `change_password` (this should metch method name).